### PR TITLE
feat: add HTML semantics

### LINKED ISSUE #1722: Apply WCAG standards to log in screen
**As a** *SPAR User* **I want** *SPAR to follow WCAG standards* **So That** *I can use assistive technologies to help me complete my work*

**Acceptance Criteria**

For the SPAR log in screen 
image

- [x] Semantic HTML structure
- [x] Ensure all images have alternative text (for the photo on the log in screen use: "Picture of spruce tree branches with needles and cones".
- [ ] Accessible forms
- [x] Keyboard navigation
- [x] ARIA (Accessible Rich Internet Applications) roles and attributes
- [ ] Focus management
- [x] Check that contrast is above 4.5:1
- [x] Tested - Use accessibility tools and screen readers during development.
   
**Additional Context**
- More information here: https://github.com/bcgov/nr-spar/wiki/Coding-for-accessibility:-Screen-readers

**Definition of Done**
- [x] Ready to Demo in Sprint Review
- [x] Does what I have made have appropriate test coverage?
- [x] Documentation and/or scientific documentation exists and can be found
- [x] Peer Reviewed by 2 people on the team
- [ ] Manual testing of all PRs in Dev and Prod
- [ ] QA from UX team member
- [ ] Merged

### PR DESCRIPTION
**Pull request by ngunner15**
# Description
Closes #1722 

#### New
- WCAG changes to the Login page

#### Changed
- Login page

#### Removed
- None

### How was this tested?
- [ ] 🧠 Not needed
- [x] 👀 Eyeball
- [ ] 🤖 Added tests
- [x] Used [Wave](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) chrome extension

###  What gif/image best describes this PR or how it makes you feel?

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://nr-spar-42-frontend.apps.silver.devops.gov.bc.ca/)
- [Backend](https://nr-spar-1742-backend.apps.silver.devops.gov.bc.ca/swagger-ui/index.html)
- [Oracle-API](https://nr-spar-1742-oracle-api.apps.silver.devops.gov.bc.ca/actuator/health)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-spar/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-spar/actions/workflows/merge.yml)

### CODE REVIEW

File: frontend/src/views/Landing/index.tsx [hunk 1]
```diff
@@ -85,6 +88,8 @@ const Landing = () => {
 
           {/* Login buttons */}
           <Button
+            type="button"
+            role="button"
```
Comment (mgaseta) [inline frontend/src/views/Landing/index.tsx:92]: Do we really need both the type and role? Shouldn't one be enough?

**Review by mgaseta — APPROVED** [review]
LGTM! Added only one question.

File: frontend/src/views/Landing/index.tsx (hunk 1 above)
Comment (ngunner15) [reply frontend/src/views/Landing/index.tsx:92]: https://github.com/bcgov/nr-spar/pull/1742/commits/5660f5b2a81353419edaf2243e7e0f8f11913a8f

Comment (ngunner15) [reply frontend/src/views/Landing/index.tsx:92]: Yes, we don't need role, only type is necessary for semantic HTML.

Comment (mgaseta) [reply frontend/src/views/Landing/index.tsx:92]: I think you removed on the other button, this one still has both :)

Comment (ngunner15) [reply frontend/src/views/Landing/index.tsx]: https://github.com/bcgov/nr-spar/pull/1742/commits/ec69f4d289d8cfe895315fcd42e4c4a232eae190

Comment (ngunner15) [reply frontend/src/views/Landing/index.tsx]: Fixed it 😄

**Review by xiaopeng0202 — APPROVED** [review]

**Review by xiaopeng0202 — APPROVED** [review]

**Pull request merged**